File: AEDS-II/verde/planoDeInternet/planoNet.cs
using System;

class Program {
    static void Main() {
    
        int plano = int.Parse(Console.ReadLine());
        int meses = int.Parse(Console.ReadLine());

        int sobra = plano;

        for (int i = 0; i < meses; i++) {
            int gasto = int.Parse(Console.ReadLine());
            sobra += plano - gasto;
        }

        Console.WriteLine(sobra);
    }
}

File: AEDS-II/verde/TP01/planoDeInternet/planoNet.c
#include <stdio.h>
#include <stdlib.h>

void lerPlano(int *plano){
    scanf("%d", plano);
}

void lerMeses(int *meses){
    scanf("%d", meses);
}

void calcularSobra(int plano, int meses){

    int sobra = plano;
    int gasto;
    
    for(int i = 0; i < meses; i++){
        scanf("%d", &gasto);
        sobra += plano - gasto;
    }
    
    printf("%d\n", sobra);
}


int main(){

    int meses;
    lerMeses(&meses);
    int plano;
    lerPlano(&plano);

    calcularSobra(meses, plano);

    return 0;
}